# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../subversion/libtool-fix.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

I know it's hardcoded but it doesn't compile without it - Aldas (Baldzius)

diff -Nur subversion-1.3.0-orig/subversion/bindings/swig/perl/native/Makefile.PL.in subversion-1.3.0/subversion/bindings/swig/perl/native/Makefile.PL.in
--- subversion-1.3.0-orig/subversion/bindings/swig/perl/native/Makefile.PL.in	2005-10-14 20:45:33.000000000 +0000
+++ subversion-1.3.0/subversion/bindings/swig/perl/native/Makefile.PL.in	2006-01-20 11:50:31.225901500 +0000
@@ -28,6 +28,8 @@
 my $apr_shlib_path_var = '@SVN_APR_SHLIB_PATH_VAR@';
 my $apr_cflags = '@SVN_APR_INCLUDES@';
 my $apu_cflags = '@SVN_APRUTIL_INCLUDES@';
+my $apr_apache_flags = `apr-1-config --cppflags`;
+chomp ($apr_apache_flags);
 
 # According to the log of r7937, the flags guarded by the conditional break
 # the build on FreeBSD if not conditionalized.
@@ -38,7 +40,7 @@
 
 my %config = (
     ABSTRACT => 'Perl bindings for Subversion',
-    INC  => join(' ',$apr_cflags, $apu_cflags, 
+    INC  => join(' ',$apr_cflags, $apu_cflags, $apr_apache_flags, 
                  " -I$swig_srcdir/perl/libsvn_swig_perl",
                  " -I$svnlib_srcdir/include",
                  " -I$swig_srcdir -g"),
